SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -333,9 +355,9 @@ - (SDLStreamHasBytesHandler)sdl_controlStreamHasBytesHandlerForAccessory:(EAAcce
 
             // Determine protocol string of the data session, then create that data session
             NSString *indexedProtocolString = [NSString stringWithFormat:@"%@%@", indexedProtocolStringPrefix, @(buf[0])];
-            dispatch_sync(dispatch_get_main_queue(), ^{
+            if (accessory.isConnected){
```

Comment:
Formatting on the `{`

Why was this added, the control stream can't send us bytes if the accessory isn't connected, right?